src/css/jwplayer/imports/logo.less

```diff
@@ -6,7 +6,7 @@
     cursor: pointer;
     pointer-events: all;
 
-    background-repeat: no-repeat;
+    background: no-repeat right;
```

Comment:
When `position` is `top-left` or `bottom-left`, the logo is not positioned correctly.